.devcontainer/Dockerfile

```diff
@@ -7,9 +7,12 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
         babeltrace python3-bt2 ros-humble-rmw-cyclonedds-cpp \
         ros-humble-cyclonedds* net-tools less python3 python3-pip \
         python3-tk software-properties-common vim babeltrace2 \
-        python3-babeltrace ros-humble-rqt-graph gh
+        python3-babeltrace ros-humble-stereo-image-proc \
```

Comment:
These changes seem unrelated to the topic of the PR.  I'd suggest making a clean (single-commit) PR with just the issue template file.